docs/archive_data_migration_to_berkeley.md

```diff
@@ -0,0 +1,470 @@
+# Migrating Mainnet Database to Berkeley
+
+Before you start planning the migration of your Mainnet database into the Berkeley version of the Mina network, read this document in entirety.
+
+## Migration apps
+
+Two applications are required to migrate the Mainnet database into the Berkeley version of the Mina network. 
+
+These applications are shipped by MF/O1 Labs:
```

Comment:
I know this is the intro, but how are these applications shipped? 

(I don't want us to call them "first migration app" and "second migration app" let's call them by name)